src/AccessibilityInsights.Actions/Actions/SaveAction.cs

```diff
@@ -86,7 +86,7 @@ private static void SaveSnapshotFromElement(int? focusedElementId, A11yFileMode
                 }
             }
 
-            var meta = new SnapshotMetaInfo(mode, SuiteFactory.GetRuleVersion(), focusedElementId, ec.DataContext.ScreenshotElementId, otherProperties);
+            var meta = new SnapshotMetaInfo(mode, RuleRunner.GetRuleVersion(), focusedElementId, ec.DataContext.ScreenshotElementId, otherProperties);
```

Comment:
I'm not sure if we still want this rule version (has always been at 1.0). I could imagine it being useful in the future so I've left it in for now.

Reply:
IMO we need since when an error is reported by a user, we want to understand which build of rules were used. 

---
In reply to: [271959148](https://github.com/Microsoft/accessibility-insights-windows/pull/256#discussion_r271959148) [](ancestors = 271959148)

Reply:
@neocln I agree, it means we need to update the version number when the rules change. We don't do that at the moment - maybe it can include the axe-windows version at some point.